tests/autoyast/installation.pm

```diff
@@ -16,6 +16,7 @@
 use strict;
 use base 'basetest';
 use testapi;
+use utils qw(clear_console);
 
```

Comment:
and to underline my argument: you will find tons of 

use Time::HiRes qw(sleep);

in our tests - but not a single sub-second sleep.

Because test writer beginners have no idea about perl and just copy&paste from other code.